common/components/whats-new-banner/whats-new-banner.yaml

```diff
@@ -11,10 +11,11 @@ params:
 examples:
   - name: default
     data:
-      contentTitle: 'Title Here'
-      contentDate: '4th March 2022'
-      contentBannerText: 'Some random banner text'
+      title: 'Title Here'
+      date: '4th March 2022'
+      body: 'Some random banner text'
   - name: withError
     data:
-      error: 'ServerError'
-      errorMessage: 'Server error occured.'
+      title: null
+      date: null
+      body: null
```

Comment:
I don't think we need this anymore since we'd always expect these to be set for this component.